thrust/thrust/detail/config.h

```diff
@@ -22,6 +22,8 @@
 #include <thrust/detail/config/config.h> // IWYU pragma: export
 #include <thrust/version.h> // IWYU pragma: export
 
+#include <cuda/__nvtx/nvtx.h>
```

Comment:
That should come after the include guard AFAIK

Reply:
which include guard?

Reply:
Oh sorry, I mean after the system header guard below, same as the version.h

Reply:
I think I still fail to understand sorry 😬. `version.h` is *before* the system header guard. Do you suggest I move `nvtx.h` below the  `#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC) #endif` block?

Reply:
@miscco is this resolved?